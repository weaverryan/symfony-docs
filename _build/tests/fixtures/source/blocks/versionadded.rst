
.. versionadded:: 4.1
    This option was introduced in Symfony 2.6 and replaces another option, which is available prior to 2.6.