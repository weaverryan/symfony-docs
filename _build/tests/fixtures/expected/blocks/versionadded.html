<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>
<div class="versionadded">
    <div>
        <span class="versionmodified">New in version 4.1: </span>
        <p>This option was introduced in Symfony 2.6 and replaces another option, which is available prior to 2.6.</p>
    </div>
</div>
</body>
</html>